android/filament-android/src/main/java/com/google/android/filament/Renderer.java

```diff
@@ -96,11 +297,67 @@ public void readPixels(
         }
     }
 
-    double getUserTime() {
+    /**
+     * Returns the time in second of the last call to {@link #beginFrame}. This value is constant
```

Comment:
"in second" => "in seconds"

I would phrase this as: `Returns a timestamp (in seconds) for the last call...`